src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/ReadPosRankSumTest.java

```diff
@@ -69,6 +70,9 @@ public static OptionalDouble getReadPosition(final GATKRead read, final int refL
         final int leadingHardClips = firstElement.getOperator() == CigarOperator.HARD_CLIP ? firstElement.getLength() : 0;
         final int trailingHardClips = lastElement.getOperator() == CigarOperator.HARD_CLIP ? lastElement.getLength() : 0;
 
+        if (offset >= cigar.getReadLength()) {
+            return OptionalDouble.empty();
```

Comment:
It's intentional.  A test was failing because correct left-alignment exposed an edge case.  I'm about to issue a PR related to `ReadPosRankSumTest` that will re-do this code.